lambdas/process-cri-callback/src/test/java/uk/gov/di/ipv/core/processcricallback/pact/passportCri/ContractTest.java

```diff
@@ -0,0 +1,260 @@
+    void testCallToDummyPassportIssueCredential(MockServer mockServer)
+            throws URISyntaxException, CriApiException {
+        // Arrange
+        var credentialIssuerConfig = getMockCredentialIssuerConfig(mockServer);
+
+        ContraIndicatorConfig ciConfig1 = new ContraIndicatorConfig(null, 4, null, null);
+        ContraIndicatorConfig ciConfig2 = new ContraIndicatorConfig(null, 4, null, null);
+
+        Map<String, ContraIndicatorConfig> ciConfigMap = new HashMap<>();
+        ciConfigMap.put("A02", ciConfig1);
+        ciConfigMap.put("A03", ciConfig2);
+
+        when(mockConfigService.getCriConfig(any())).thenReturn(credentialIssuerConfig);
+        when(mockConfigService.getCriPrivateApiKey(any())).thenReturn(PRIVATE_API_KEY);
+        when(mockConfigService.getContraIndicatorConfigMap()).thenReturn(ciConfigMap);
+
+        var verifiableCredentialJwtValidator =
+                new VerifiableCredentialJwtValidator(mockConfigService);
+
+        // We need to generate a fixed request, so we set the secure token and expiry to constant
+        // values.
+        var underTest =
+                new CriApiService(
+                        mockConfigService,
+                        mockSigner,
+                        mockSecureTokenHelper,
+                        Clock.fixed(Instant.parse("2099-01-01T00:00:00.00Z"), ZoneOffset.UTC));
+
+        // Act
+        var verifiableCredentialResponse =
+                underTest.fetchVerifiableCredential(
+                        new BearerAccessToken("dummyAccessToken"),
+                        new CriCallbackRequest(
+                                "dummyAuthCode",
+                                credentialIssuerConfig.getClientId(),
+                                "dummySessionId",
+                                "https://identity.staging.account.gov.uk/credential-issuer/callback?id=ukPassport",
+                                "dummyState",
+                                null,
+                                null,
+                                "dummyIpAddress",
+                                "dummyFeatureSet"),
+                        new CriOAuthSessionItem(
+                                "dummySessionId",
+                                "dummyOAuthSessionId",
+                                "dummyCriId",
+                                "dummyConnection",
+                                900));
+
+        verifiableCredentialResponse
+                .getVerifiableCredentials()
+                .forEach(
+                        credential -> {
+                            try {
+                                verifiableCredentialJwtValidator.validate(
+                                        credential, credentialIssuerConfig, TEST_USER);
+                            } catch (VerifiableCredentialException e) {
+                                throw new RuntimeException(e);
+                            }
+                        });
```

Comment:
This is a bit of a fuzzy line, but I think I'd expect these assertions to at least cover things like checking that this has the claims we'd expect from the passport CRI.
e.g. assuming that `CREDENTIAL_ATTRIBUTES_2` is realistic then we should be checking that there are name, DoB, and address claims present with some values in them.

Reply:
@blakeyp Do you think it is worth having a higher level of validation in the test than in the actual codebase? 

If we want to validate to a higher level in the test, then it feels like we should spin out another ticket to increase the validation in the code itself. (I think Joe was also looking at making a ticket for this)